development-guide.md

```diff
@@ -49,6 +49,25 @@ Partners should not add the proto files to their repos. Proto files should be pu
 - Partner code should handle any source/destination errors, retry any transient errors internally without deferring them to Fivetran.
 - Partner code should use [gRPC built-in error mechanism](https://grpc.io/docs/guides/error/#language-support) to relay errors instead of throwing exceptions and abruptly closing the connection.
 
+### User Tasks
+
+- Partners can throw alerts on dashboard to notify customers when there is a problem in their Fivetran account, such as a broken connector or an incomplete sync, and tells you how to resolve that problem.
```

Comment:
`Partners can throw alerts on the Fivetran dashboard to notify customers when there is a problem in their connector.`
And for the second line: the connector breaks because partners are throwing tasks and not vice versa.

Reply:
I corrected the first statement. According to me the examples are fine

Reply:
We are the ones who break the connector if it is not working as expected by throwing the task. And that would cause incomplete sync.
e.g. Inappropriate value for a column (bad source), in this situation we break the connector intentionally and not that the connector broke so we are asking partners to throw a task. 

Updated doc can be: 
`Partners can throw alerts on the Fivetran dashboard to notify customers about potential issues with their connector. These issues could include bad source data or connection problems with the source itself. The alerts can also provide guidance to customers on how to resolve the problem.`